### PR TITLE
crowbar-pacemaker: Add ability to set custom attributes on nodes

### PR DESCRIPTION
**Pull request by vuntz**
This will be useful to set the OpenStack-role attribute.

Note that this contains https://github.com/crowbar/crowbar-ha/pull/55 -- only the last commit is different here.

### CODE REVIEW

File: chef/cookbooks/crowbar-pacemaker/recipes/remote.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# The parent recipe to be run on all remote nodes.
+
+# Figure out which cluster we're tied to, so that we can find the founder
+# and hence the authkey.
+node[:corosync][:cluster_name] = CrowbarPacemakerHelper.cluster_name(node)
```
Comment (jsuchome) [inline chef/cookbooks/crowbar-pacemaker/recipes/remote.rb]: Isn't `node.save` needed?

**Comment by jsuchome** [comment]
+1 for the last commit

**Comment by vuntz** [comment]
I moved one part of the last commit to https://github.com/crowbar/crowbar-ha/pull/62 so we can unblock other PR faster.

**Comment by vuntz** [comment]
https://github.com/crowbar/crowbar-ha/pull/62 got merged, and I moved the remaining bit back to https://github.com/crowbar/crowbar-ha/pull/55 so this PR isn't needed anymore.

**Pull request closed without merging**